libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -165,9 +144,17 @@ bool AP_MotorsHeli_Single::init_outputs()
         _swash_servo_2 = SRV_Channels::get_channel_for(SRV_Channel::k_motor2, CH_2);
         _swash_servo_3 = SRV_Channels::get_channel_for(SRV_Channel::k_motor3, CH_3);
         _yaw_servo = SRV_Channels::get_channel_for(SRV_Channel::k_motor4, CH_4);
-        _servo_aux = SRV_Channels::get_channel_for(SRV_Channel::k_motor7, CH_7);
-        if (!_swash_servo_1 || !_swash_servo_2 || !_swash_servo_3 || !_yaw_servo || !_servo_aux) {
-            return false;
+        if (_tail_type == AP_MOTORS_HELI_SINGLE_TAILTYPE_DIRECTDRIVE_VARPITCH) {
+            //_servo_aux = SRV_Channels::get_channel_for(SRV_Channel::k_heli_tail_rsc, CH_7);
```

Comment:
If this is commented on purpose, let's take it out, we don't like dead code to stay in master.

Reply:
Sorry. I thought that had deleted it in the commits that I made for the copter 3.5 PR. I will fix it.

Reply:
@oxinarf should I squash the commit fixing this dead code with the other commit and then force the push? Will that mess things up in the PR?

Reply:
@oxinarf should I squash the commit fixing this dead code with the other commit and then force the push? Will that mess things up in the PR?

Reply:
@bnsgeyer Yes, squash and force push should work perfectly fine.